packages/interactive_media_ads/ios/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
@stuartmorgan @jmagman Are there resources for me to find what I would need to add to this? This plugin is wrapping: https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side

Reply:
As long as the plugin isn't directly calling any Required Reason APIs (I would be very surprised if it were) and isn't itself sending data to any servers (it definitely isn't if it's just a direct wrapper), it would not need to declare anything. See https://github.com/flutter/packages/blob/main/packages/file_selector/file_selector_ios/ios/Resources/PrivacyInfo.xcprivacy for example.

The requirements above are specifically about the code *in the plugin*, not anything the wrapped SDK might be doing.